CHANGELOG.next.asciidoc

```diff
@@ -524,6 +524,11 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add Kafka JMX metricsets. {pull}14330[14330]
 - Add metrics to envoyproxy server metricset and support for envoy proxy 1.12. {pull}14416[14416] {issue}13642[13642]
 - Add module for ActiveMQ. {pull}14788[14788]
+- Release kubernetes modules `controllermanager`, `scheduler`, `proxy`, `state_cronjob` and `state_resourcequota` as GA. {pull}14584[14584]
```

Comment:
:grimacing:

Reply:
@sayden Thanks for catching it! Vacation made me forgot about changelog lol